documentation/modules/configuring/proc-config-mirrormaker2-replication.adoc

```diff
@@ -111,38 +111,42 @@ spec:
         replication.factor: 1 <20>
         offset-syncs.topic.replication.factor: 1 <21>
         sync.topic.acls.enabled: "false" <22>
-        replication.policy.separator: "" <23>
-        replication.policy.class: "io.strimzi.kafka.connect.mirror.IdentityReplicationPolicy" <24>
-    heartbeatConnector: <25>
+        refresh.topics.interval.seconds: 60 <23>
+        replication.policy.separator: "" <24>
+        replication.policy.class: "io.strimzi.kafka.connect.mirror.IdentityReplicationPolicy" <25>
```

Comment:
To make this example valid, you have to also add this option to the `checkpointConnector.config`

Reply:
Added. I've left the description here, but expanded: _To configure topic offset synchronization, this property must also be set for the `checkpointConnector.config`._

Also changed the concepts: _Add the `IdentityReplicationPolicy` to the source and checkpoint connector configuration so that topics in the target cluster retain their original names._